### PR TITLE
[WIP] shop_url parameter

### PR DESCRIPTION
**Pull request by rfrancel**
shop_url as parameter with possible values:
dev
int
prod
puzzle (http://shop-shopbgdi.openshift.puzzle.ch)

### CODE REVIEW

File: src/index.mako.html [hunk 1]
```diff
@@ -758,6 +758,22 @@
           api_url = my_host;
         }
 
+        var my_shop_url = getParam('shop_url');
```
Comment (oterral) [inline src/index.mako.html]: Please avoid to name a variable with `my****` and use camelCase syntax `shopUrl`.

**Comment by oterral** [comment]
thx @rfrancel, just a minor comment.

**Comment by rfrancel** [comment]
Thanks for your comments @oterral I changed it for the api parameters as well, adn I am redeploying the branch: https://mf-geoadmin3.dev.bgdi.ch/teo_shop

**Comment by gjn** [comment]
Please squash and merge. Thanks alot!

**Pull request merged**